Marlin/example_configurations/K8200/files by Vellemann/Configuration.h

```diff
@@ -0,0 +1,480 @@
+#ifndef CONFIGURATION_H
```

Comment:
@thinkyhead The files were thought as a reference for everybody who would search for bugs in my rebased version. They should never stay in sync because they are horribly outdated now, which is one reason I ported the configs to the actual dev branch.

I will try your proposal and make a branch in my Marlin with a link in the examples readme pointing on it.

Thanks a lot for your advice and your time!